cookbook/caching-with-varnish.rst

```diff
@@ -211,6 +211,106 @@ from ``symfony`` to ``varnish`` and set the address of your varnish server
                 enabled: true
                 servers: ['localhost:80']
 
+Using XKey
+----------
+
+Xkey is an efficient way to invalidate Varnish cache entries based on tagging. But it is not integrated
+by default into varnish.
+
+To be able to use it you have to install ``varnish-modules``:
+
+.. code-block:: bash
+
+    apt-get install varnish-modules
+
+Or build it from sources:
```

Comment:
would just link to the official site that describes how to do this. this will get out of date and we wont notice otherwise